index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Romeo Lager - Cloud Engineer Apprentice </title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    </div>
+    <header>
+        <h1>Romeo Lager</h1>
+        <nav>
+            <ul>
+                <li><a href="#home">Home</a></li>
+                <li><a href="#bio">Bio</a></li>
+                <li><a href="#experience">Experience</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+            </ul>
+        </nav>
+    </header>
+<main>
```

Comment:
great use of semantic tags throughout this page to increase accessibility! see if there are any others that you could use: https://www.w3schools.com/html/html5_semantic_elements.asp